src/components/ha-service-picker.js

```diff
@@ -4,18 +4,20 @@ import { PolymerElement } from "@polymer/polymer/polymer-element";
 import "./ha-combo-box";
 
 import LocalizeMixin from "../mixins/localize-mixin";
+import EventsMixin from "../mixins/events-mixin";
 
 /*
  * @appliesMixin LocalizeMixin
  */
-class HaServicePicker extends LocalizeMixin(PolymerElement) {
+class HaServicePicker extends EventsMixin(LocalizeMixin(PolymerElement)) {
   static get template() {
     return html`
       <ha-combo-box
         label="[[localize('ui.components.service-picker.service')]]"
         items="[[_services]]"
         value="{{value}}"
         allow-custom-value=""
+        on-change="_fireChanged"
```

Comment:
Remove this. `value` is two-way data binded and the property is set with `notify: true`, so just listen for `value-changed` events.